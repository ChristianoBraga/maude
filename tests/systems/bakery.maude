set show timing off .

load ../../contrib/systems/bakery.maude

mod BAKERY-META-LEVEL is
   protecting BAKERY .
   protecting META-LEVEL .
endm

search [,2] s 0 ; s 0 ; [idle] [idle] =>* C:Conf .
