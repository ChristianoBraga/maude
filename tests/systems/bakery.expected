==========================================
search [, 2] in BAKERY-META-LEVEL : s ; s ; [idle] [idle] =>* C:Conf .

Solution 1 (state 0)
states: 1  rewrites: 0
C:Conf --> s ; s ; [idle] [idle]

Solution 2 (state 1)
states: 2  rewrites: 1
C:Conf --> (s s) ; s ; [idle] [wait(s)]

Solution 3 (state 2)
states: 3  rewrites: 2
C:Conf --> (s s s) ; s ; [wait(s)] [wait(s s)]

Solution 4 (state 3)
states: 4  rewrites: 3
C:Conf --> (s s) ; s ; [idle] [crit(s)]

No more solutions.
states: 4  rewrites: 3
Bye.
